src/applications/personalization/profile360/components/PaymentInformationEditModalError.jsx

```diff
@@ -3,23 +3,29 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 // possible values for the `key` property on error messages we get from the server
 const ACCOUNT_FLAGGED_FOR_FRAUD_KEY = 'cnp.payment.flashes.on.record.message';
+const GENERIC_ERROR_KEY = 'cnp.payment.generic.error.message';
 const INVALID_ROUTING_NUMBER_KEY =
   'payment.accountRoutingNumber.invalidCheckSum';
-const GENERIC_ERROR_KEY = 'cnp.payment.generic.error.message';
+const PAYMENT_RESTRICTIONS_PRESENT_KEY =
+  'payment.restriction.indicators.present';
+const ROUTING_NUMBER_FLAGGED_FOR_FRAUD_KEY =
+  'cnp.payment.routing.number.fraud.message';
 
 function FlaggedAccount() {
   return (
     <>
       <p>
         We’re sorry. You can’t change your direct deposit information right now
-        because we’ve locked your account. We do this to protect your bank
-        account information and prevent fraud when we think there may be a
-        security issue.
+        because we’ve locked the ability to edit this information. We do this to
```

Comment:
The wording of this error message has changed as requested here https://github.com/department-of-veterans-affairs/va.gov-team/issues/2352#issuecomment-543371597 and here
https://github.com/department-of-veterans-affairs/va.gov-team/issues/2352#issuecomment-543361365